Makefile

```diff
@@ -45,6 +45,11 @@ setup-db:
 default-config:
 	cp -i build/app/config.dev.json app/config
 
+download-assets:
+	curl -L https://gitlab.com/api/v4/projects/fun-tech%2ffundraising-app-frontend/jobs/artifacts/$(ASSET_BRANCH)/download?job=build-artifacts > var/assets.zip
```

Comment:
executing `$ make setup` gives me a permission denied error when it tries to create `var/assets.zip`, execution aborts